### PR TITLE
Ops genie description text

### PR DESCRIPTION
**Pull request by kelseiv**
Add new details configuration option to OpsGenie v2 event handler.

### CODE REVIEW

File: content/kapacitor/v1.5/event_handlers/opsgenie/v2.md [hunk 1]
```diff
@@ -57,6 +58,9 @@ Valid values include:
   * `close` - Close the alert.
   * custom -  Use the `.RecoveryAction()` method to specify the `recovery_action` in a TICK script.
 
+#### `details`
+If `true`, all alerts use their event details as the OpsGenie alert description. If `false`, event details are encoded as description.
```
Comment (kelseiv) [inline content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:62]: Perhaps we can clarify how event details are encoded as description (being that's the default)?

Comment (sanderson) [reply content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:63]: Yeah, "event details are encoded as description" isn't clear at all. I'm not even sure what that means.

**Review by sanderson — COMMENTED** [review]
What is the difference between "[using] event details as the OpsGenie alert description" and "event details are encoded as description"? I don't fully understand the difference in behavior.

File: content/kapacitor/v1.5/event_handlers/opsgenie/v2.md (hunk 1 above)
Comment (docmerlin) [reply content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:63]: @sanderson @kelseiv It means the details field, gets sent as the OpsGenie alert description field.

**Review by docmerlin — APPROVED** [review]

File: content/kapacitor/v1.5/event_handlers/opsgenie/v2.md (hunk 1 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:63]: Thank you, @docmerlin, to be sure, we're describing this details field that we set to true or false, which in turn sends another details field as the OpsGenie alert description field? 

Want to be sure on the exact name/location of this field--would "event details field" or "alert details field" be more accurate? thanks!

Comment (docmerlin) [reply content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:63]: With would be fine.  Alert might make more sense for some people? Shrug.

Comment (kelseiv) [reply content/kapacitor/v1.5/event_handlers/opsgenie/v2.md:63]: do we not use a specific term in our product?

**Pull request merged**